tests/test_modeling_common.py

```diff
@@ -386,7 +388,7 @@ def _create_and_check_torchscript(self, config, inputs_dict):
 
             try:
                 if model.config.is_encoder_decoder:
-                    model.config.use_cache = False  # TODO: this should be deleted after bug #7474 is solved
+                    model.config.use_cache = False  # FSTM still requires this hack -> FSTM should probably be refactored similar to BART afterward
```

Comment:
yeah, not really using cache in FSMT - I wasn't sure what to do with it there. Really was trying to making things work and some parts weren't quite needed but seemed to be required by common tests and components. Looking forward to fsmt refactoring! yay!